website/docs/api-backends.md

```diff
@@ -927,6 +927,47 @@ Backpropagate the Swish activation
 | `inplace`   | <tt>bool</tt>     | If `True`, the `dY` array is modified in place. |
 | **RETURNS** | <tt>FloatsXd</tt> | The gradient of the input.                      |
 
+### Ops.dish {#dish tag="method"}
```

Comment:
```suggestion
### Ops.dish {#dish tag="method" new="8.1.1"}
```
(assuming this will be in 8.1.1 and not 8.2 - though I'm not sure we know this already.

Reply:
Fixed!